internal/bminventory/inventory.go

```diff
@@ -4898,6 +4898,13 @@ func (b *bareMetalInventory) UpdateInfraEnvInternal(ctx context.Context, params
 		return nil, common.NewApiError(http.StatusNotFound, err)
 	}
 
+	if params.InfraEnvUpdateParams.ImageType == models.ImageTypeMinimalIso && (infraEnv.CPUArchitecture == common.S390xCPUArchitecture || infraEnv.CPUArchitecture == common.PowerCPUArchitecture) {
```

Comment:
minimal ISO is supported for PowerCPU architecture

Reply:
OK, So we should focus on only S390X

Reply:
yes

Reply:
Do we want to list the archs we support or the archs we don't support (will affect future archs)?

Reply:
Do we have a place where the supported archs by platform are enumerated at this time?

Reply:
Actually you should probably use the information from [this PR](https://github.com/openshift/assisted-service/pull/5101)

Reply:
It looks like the PR lists the unsupported arch's here

https://github.com/openshift/assisted-service/pull/5101/files#diff-9b84f395a9312d62831dda2f04163c4e99cb307046e2a0007b1b3bc005a7d3dcR570-R586

I don't see anything that lists supported arch's, I reckon "supported arch's" should also be covered by the PR if we intend to list them as it will keep things neat.

Reply:
What are you suggesting in this comment, 

It seems to just be around how we communicate an unsupported/supported architecture. I honestly think that what we already have is uncomplicated and sufficient.

WDYT?